ts/components/wallet/PaymentBannerComponent.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * This component displays a summary on the transaction.
+ * Used for the screens from the identification of the transaction to the end of the procedure.
+ */
+
+import { Text, View } from "native-base";
+import * as React from "react";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../i18n";
+import { WalletStyles } from "../styles/wallet";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+export default class PaymentBannerComponent extends React.Component<Props> {
+  public render(): React.ReactNode {
+    const NOMEAVVISO = "Tari 2018";
```

Comment:
I suggest to define a specific type of this kind of content, as we are going to provide the same data to multiple components is better to share the object